packages/jest-editor-support/src/parsers/babylon_parser.js

```diff
@@ -162,7 +202,9 @@ export const parse = (file: string): BabylonParserResult => {
     }
   };
 
-  searchNodes(ast['program'], file);
+  if (ast) {
+    searchNodes(ast.program, file);
+  }
```

Comment:
I was able to run vscode-jest without `ast` error... if you can reproduce the issue then we can just leave the `ast` check in but throw exception for null; otherwise, it should be safe to remove the `ast` check.